src/book/03-guides/04-general/01-learning-resources.mdx

```diff
@@ -56,7 +59,7 @@ The C++ programming language is used in the main robot codebase. Modern C++ and
 </details>
 
 <details>
-<summary>NUClear</summary>
+<summary> NUClear </summary>
```

Comment:
Adding a space to these does nothing - if you want to increase the space on either side you'll need to add padding to the summary jsx component. Pls revert these changes 🙏

Reply:
Yes I thought it was strange that there was ignored whitespace in other sections of the document and thought it was for readability initially.

I'll get that corrected and merged in (might also be worth double checking it doesn't appear anywhere else in nubook with some regex to ensure this issue isn't run into again in future)